WalletWasabi/WabiSabi/Models/DecompositionAlgs/Coin.cs

```diff
@@ -0,0 +1,40 @@
+using NBitcoin;
+using System;
+
+namespace WalletWasabi.WabiSabi.Models.DecompositionAlgs
+{
+	public record Coin : IComparable
+	{
+		public Money Amount { get; init; } = Money.Zero;
+		public const int OutputVbytes = 31;
+		public const decimal InputVbytes = 68.5m;
+
+		private int HashCode { get; } = new Guid().GetHashCode();
+
+		public Money EffectiveValue(FeeRate feeRate)
```

Comment:
This could be called something like `AmountAfterFees` or `AvailableAmount`. `EffectiveValue` makes reference to something too much fuzzy from my perspective.

Reply:
fwiw core uses the same name for this concept: https://github.com/bitcoin/bitcoin/search?q=%22effective+value%22

Reply:
Removed Coin class.